templates/index.html

```diff
@@ -14,200 +14,221 @@
 <script defer src="{{ versioned_static('js/osselector.js') }}"></script>
 
 <main id="main-content">
-
-  <section class="p-strip--header is-dark">
+  <section class="p-strip--suru is-dark">
     <div class="row u-vertically-center u-equal-height">
-      <div class="col-8">
-        <h1>
-          Multi-node OpenStack for workstations and edge / IoT
-        </h1>
-        <p class="p-heading--four u-sv3">
-          Install and run OpenStack on Linux in minutes. Made for developers and great for edge, IoT, and appliances.
-        </p>
+      <div class="col-7">
+        <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
+        <p class="p-heading--four"> The most straightforward OpenStack ever. </p>
         <p>
-          <a class="p-button--positive is-inline" href="#get-started">Get started</a><span style="padding-left: .5rem;">for Linux, Windows or macOS</span>
+          <a class="p-button--positive" href="#get-started">Try MicroStack</a> &nbsp;and break the ice with OpenStack today
         </p>
       </div>
-      <div class="col-4 u-align--center u-hide--small">
-        <img src="https://assets.ubuntu.com/v1/5a6af817-OpenStack-Logo-Vertical-white.svg" width="250"
-          alt="" />
+      <div class="col-5 u-align--center u-hide--small u-embedded-media">
+        <iframe class="u-embedded-media__element" width="560" height="315" src="https://www.youtube.com/embed/fNk7AfxNggc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
       </div>
-    </div>
   </section>
-
   <section class="p-strip">
     <div class="row">
       <div class="p-notification--caution">
         <p class="p-notification__response">
-          <span class="p-notification__status">Note:</span> MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>.
+          <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>. For information about commercial support for MicroStack, <a class="p-link--external" href="https://ubuntu.com/openstack/contact-us">contact Canonical</a>.
         </p>
       </div>
     </div>
     <div class="row">
-      <div class="col-8">
-        <h2>OpenStack in a snap</h2>
-        <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+      <div class="col-8" style="margin-top: 2rem;">
+        <h2>Straightforward installation</h2>
+        <h4>Fully functional OpenStack in 2 commands and ~20 minutes.</h4>
```

Comment:
I know you didn't change this but we shouldn't jump from a h2 to a h4, you can add the classname "p-heading--4" to a h3 so it remains semantically correct with also the correct style.
```suggestion
        <h3 class="p-heading--4">Fully functional OpenStack in 2 commands and ~20 minutes.</h4>
```

Reply:
Didn't know that we shouldn't jump from a h2 to a h4. Thanks! Fixed. 👍